TestAssets/ProjectModelServer/DthUpdateSearchPathSample/home/src/MainProject/project.json

```diff
@@ -1,8 +1,10 @@
 {
   "dependencies": {
-    "Newtonsoft.Json": "6.0.8"
+    "Newtonsoft.Json": "8.0.3"
```

Comment:
FYI - if you update this to "9.0.1" you should need any `imports` in your netstandard1.5 AFAIK.